### DISCUSSION TITLE
Unhelpful Error Message: Load Failed

### DISCUSSION THREAD

**Discussion by TheWorfer27** [Pryaxis/TShock#1983] [General]
I'm attempting to upgrade to tshock 1.4 for my build server. I've moved the plugins into a non-tshock folder so I can put them back if this works. I didn't move the ones I didn't download, such as MySql.Data.Dll. When I attempt to start the server, it allows me to pick a world, after which it will load the world and then say "Load Failed: No Backup Found". I made a backup of the world, and now it simply tells me "Load Failed". I'm not sure where to go with this error, as it doesnt tell me what the issue is. My server log tells me this: 
[06/03/20 09:22:50] [Server API] Verbose: TerrariaApi - Server v2.1.0.0 started.
[06/03/20 09:22:50] [Server API] Verbose: 	Command line: "C:\Users\ascot\Downloads\Tshock From 1.3.5\TerrariaServer.exe" 
[06/03/20 09:22:50] [Server API] Verbose: 	OS: Microsoft Windows NT 6.2.9200.0 (64bit: True)
[06/03/20 09:22:50] [Server API] Verbose: 	Mono: False

Any help would be appreciated, thank you.

**Comment by TheWorfer27** [comment]
Additional Info: The world I'm attempting to load is in 1.4, though it has no unique 1.4 tiles (echo blocks, etc). My TerrariaServer application says 1.3.5.3 at the top, so that may be the cause of the issue. If it is, does anyone know how to make it instead be at version 1.4?

**Comment by hakusaro** [comment]
It sounds like you didn't update properly? Overwrite all server files from TShock for 1.3 with the latest files from the latest pre-release.
